app/views/maintainers/edit.html.erb

```diff
@@ -0,0 +1,27 @@
+<th class="container">
+  <h2>Add Maintainer for <strong><%= @project.title %></strong></h2>
+  <table class="table table-striped">
+    <thead><tr>
+      <th scope="col">#</th>
+      <th scope="col">Name</th>
+      <th scope="col">Add or Remove</th>
+    </tr></thead>
+    <tbody>
+      <% User.all.each do |user| %>
+      <tr><th scope="row"><%= user.id %></th>
+          <td><%= user.first_name %></td>
+          <td>
+            <% if @project.maintainers.include?(user) %>
```

Comment:
Ha tudjuk, akkor ezt is szuper lenne a kontrollerakcióban lévő query-be átmozgatni.